server.js

```diff
@@ -4458,6 +4458,75 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Bitbucket Pipelines integration.
+camp.route(/^\/bitbucket\/pipelines\/([^/]+)\/([^/]+)(?:\/(.+))?\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var user = match[1];  // eg, atlassian
+  var repo = match[2];  // eg, adf-builder-javascript
+  var branch = match[3] || 'master';  // eg, development
+  var format = match[4];
```

Comment:
When adding new code could you use the ES6 style `let` and `const` rather than `var` please.

Reply:
Will do, thanks. I tried to stick to the existing style, but happy to use ES6.